src/appleseed/foundation/math/sampling/qmcsamplingcontext.h

```diff
@@ -55,6 +57,7 @@ namespace foundation
 // A sampling context featuring:
 //
 //   - deterministic sampling based on Halton sequences
+//   - deterministic sampling based on modified Chiu et al’s multi-jittered sampling method
```

Comment:
Unless I'm mistaking, `’` is not a standard quote, and the file is no longer encoded in ANSI. Use a standard quote. (Are you using Sublime Text by any chance?)

Reply:
No, vim ^^